code/modules/power/engines/supermatter/supermatter_event.dm

```diff
@@ -136,6 +147,7 @@
 //A class events
 /datum/supermatter_event/alpha_tier
 	threat_level = SM_EVENT_THREAT_A
+	duration = 10 SECONDS
```

Comment:
I didnt add durations here because some of these events are instant, like A-1 and A-2, so they dont really need an announcement saying they're over

Reply:
Durations were added as otherwise the event will continue to be "active" for the rest of the round!